sentry/src/test/resources/json/user.json

```
@@ -3,6 +3,12 @@
     "id": "efb2084b-1871-4b59-8897-b4bd9f196a01",
     "username": "60c05dff-7140-4d94-9a61-c9cdd9ca9b96",
     "ip_address": "51d22b77-f663-4dbe-8103-8b749d1d9a48",
+    "name": "c8c60762-b1cf-11ed-afa1-0242ac120002",
+    "geo": {
+        "city": "0e6ed0b0-b1c5-11ed-afa1-0242ac120002",
+        "country_code": "16717632-b1c5-11ed-afa1-0242ac120002",
```

`low`: maybe use a better value here
```suggestion
        "country_code": "JP",
```